revapi-reporter-text/src/test/java/org/revapi/reporter/text/TextReporterTest.java

```diff
@@ -111,7 +117,8 @@ public void testCustomTemplate() throws Exception {
 
             String expected = "old1 VS new1\nold2 VS new2\n";
 
-            assertEquals(expected, out.toString());
+            assertEquals(STANDARDIZE_LINE_BREAKS.matcher(expected).replaceAll(""),
```

Comment:
Similar issue as the JSON string problem but with line endings being different. Changed this test to be agnostic of system line endings by stripping out all `\r\n` / `\n` characters before comparing.